source/core/slang-io.cpp

```diff
@@ -48,8 +48,17 @@ namespace Slang
 /* static */ SlangResult File::remove(const String& fileName)
 {
 #ifdef _WIN32
-    // https://docs.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-deletefilea
-    if (DeleteFileA(fileName.getBuffer()))
+    // Convert to wide string for Unicode support
+    int wideLen = MultiByteToWideChar(CP_UTF8, 0, fileName.getBuffer(), -1, nullptr, 0);
```

Comment:
use `fileName.toWString()` to get `wchar_t*`, don't call windows API directly to do the translation.